infinite-leases.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+NETWORK_NAME=${CLUSTER_NAME}bm
+FILENAME=/var/lib/libvirt/dnsmasq/${NETWORK_NAME}.hostsfile
+
+# We need to keep running even after setting the infinite leases because
+# they are overwritten multiple times in the deployment process.
+while :
+do
+    if grep -q -v infinite $FILENAME
+    then
+        sudo perl -pi -e 's/(.*?)(,infinite|)$/\1,infinite/' $FILENAME
```

Comment:
adding this to the hostsfile means that we cant restart the libvirt network (I guess the lease-init script can't parse it), maybe we don't care but I thought it was worth pointing out.

```
[derekh@u07 dev-scripts]$ sudo virsh net-start ostestbm
error: Failed to start network ostestbm
error: internal error: Child process (VIR_BRIDGE_NAME=ostestbm /usr/sbin/dnsmasq --conf-file=/var/lib/libvirt/dnsmasq/ostestbm.conf --leasefile-ro --dhcp-script=/usr/libexec/libvirt_leaseshelper) unexpected exit status 11: 
dnsmasq: lease-init script returned exit code 1
[derekh@u07 dev-scripts]$ kill: not enough arguments
Added infinite leases to /var/lib/libvirt/dnsmasq/ostestbm.hostsfile

```
Also the scripts that generates the clouds.yaml no longer works, as it can't get the host IP addresses with virsh
```
[derekh@u07 dev-scripts]$ virsh net-dhcp-leases ostestbm
error: failed to get network 'ostestbm'
error: Network not found: no network with matching name 'ostestbm'
```

Reply:
I guess if its not on by default then its not a big problem, and it'll have to be sorted somehow before getting switched on.